src/PowerSimulations.jl

```diff
@@ -84,8 +84,8 @@ export StoredEnergy
 
 #operation_models
 export GenericOpProblem
-#export UnitCommitment
-#export EconomicDispatch
+export UnitCommitmentProblem
+export EconomicDispatchProblem
```

Comment:
I'm not sure how these got un-commented out.  Must be a merge error.  What should it be @jd-lara ?

Reply:
I uncommented those, they had the wrong name.